tests/Microsoft.Identity.Web.Test/Resource/AadIssuerValidatorTests.cs

```diff
@@ -100,11 +98,22 @@ public void Validate_NullOrEmptyTenantId_ThrowsException()
             Assert.Equal(IDWebErrorMessage.TenantIdClaimNotPresentInToken, exception.Message);
         }
 
-        [Fact]
-        public void Validate_IssuerMatchedInValidIssuer_ReturnsIssuer()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void Validate_IssuerMatchedInValidIssuer_ReturnsIssuer(bool shortTidClaim)
```

Comment:
Instead of a bool, can pass the claim name itself as a parameter, and then into a constructor.

Reply:
yes. nice suggestion.